packages/spor-react/src/alert/AlertIcon.tsx

```diff
@@ -81,4 +84,10 @@ const texts = createTexts({
     sv: "Servicemeddelande",
     en: "Service message",
   },
+  "global-deviation": {
+    nb: "Dropsmelding",
+    nn: "Dropsmelding",
+    sv: "Dropsmelding",
+    en: "Dropsmelding",
```

Comment:
"Dropsmelding" er et internt navn hos oss, så ut mot kunde burde nok dette være noe annet... Kanskje ta en prat med Monica f.eks? Noen forslag (uten at jeg vet hva som blir riktig som aria-label for ikonet: "Avvik", "Togavvik" eller "Advarsel" (samme som outline versjonen)?